daemon/cdi_other.go

```diff
@@ -0,0 +1,11 @@
+//go:build !linux
+// +build !linux
+
+package daemon
+
+import "github.com/pkg/errors"
+
+// registerCdiDriver is a no-op on non-Linux platforms.
+func registerCdiDriver() error {
+	return errors.New("CDI is not supported on this platform")
```

Comment:
I think this should use `errdefs.NotImplemented`.

Reply:
Thanks. Will update.